test/runit/tests/heatmap_downloadData.R

```diff
@@ -0,0 +1,35 @@
+source("web-app/HeimScripts/heatmap/downloadData.R")
+
+.setUp <- function() {
+  origDirectory <<- getwd()
```

Comment:
<<- does not save in global environment, but in the parent scope. assign("origDirectory", getwd(), envir = .GlobalEnv) may look uglier but you are always sure that it writes to the global scope.